packages/website/pages/docs/guides/run-a-node.mdx

```diff
@@ -40,14 +40,22 @@ Then, open the `.env` file in your preferred text editor:
   <Tab>```sh notepad .env ```</Tab>
 </Tabs>
 
-Finally, set the following environment variables to configure your node: 
+Finally, set the following environment variables:
+  - L1_ENDPOINT_HTTP
+  - L1_ENDPOINT_WS
+
+> **Note:** You can get a Sepolia L1 endpoint from a few places, [Alchemy](https://www.alchemy.com/) is one popular example.
+
+### Enable your node as a prover (optional)
```

Comment:
Shall we also put a reminder here that only the first prover can get the reward, and others will be rejected by the protocol smart contract?

Reply:
done